src/index.js

```diff
@@ -16,7 +17,21 @@ const io = socketIO(http);
 app.use(cors());
 app.get("/s/:shortUrl", shortUrlStatic);
 
-app.use(egoToken({ required: true, egoURL }));
+app.use(bodyParser.json());
```

Comment:
I'd thought that you'd mentioned that `params` were in `formData` of the request yesterday.. maybe I misunderstood? This is working locally though right?